dataland-frontend/src/components/forms/parts/fields/FormFieldProps.ts

```diff
@@ -81,3 +89,37 @@ export const MultiSelectFormProps = Object.assign(deepCopyObject(FormFieldPropsW
     required: false,
   },
 }) as Readonly<ComponentPropsOptions>;
+
+export const DataPointFormFieldProps = Object.assign(deepCopyObject(YesNoFormFieldProps as ObjectType), {
+  dataTest: {
+    type: String,
+  },
+  reportsName: {
+    type: Array<string>,
+  },
+  placeholder: {
+    type: String,
+  },
+  options: {
+    type: Array,
+  },
+}) as Readonly<ComponentPropsOptions>;
```

Comment:
Also not really necessary to add the props to this file here. You could paste it directly to the component, since it is only used there.

Reply:
I think currently they are also imported in the uploadReports component. So i would keep them here